filter_plugins/insert_kernel_modules.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+import cpuinfo
+
+
+def get_cpu_info():
+    info = cpuinfo.get_cpu_info()
+    return info["family"], info["model"]
+
+
+def is_in_range(value, start, end):
+    return start <= value <= end
+
+
+def match_single_constraint(value, constraint):
+    # Single value specified
+    if isinstance(constraint, int):
+        if value == constraint:
+            return True
+
+    # List of ranges specified
+    elif isinstance(constraint, list):
+        for item in constraint:
+            range = item.get("range")
+            if not range:
+                raise ValueError(
+                    "Constraints needs to be int or list of ranges!"
+                )
+            if is_in_range(value, range.get("start", 0), range.get("end", 255)):
```

Comment:
Do we need to cap the range to 255? It looks like unnecessary conctraint to me.

Reply:
Addressed in ff0daa1 . The  limits of 0 and 255 were replaced by -inf and inf